src/path.ml

```diff
@@ -425,4 +425,12 @@ let change_extension ~ext t =
   let t = try Filename.chop_extension t with Not_found -> t in
   t ^ ext
 
+let get_extension t =
+  try
+    let t' = Filename.chop_extension t in
+    let l = String.length t' in
+    String.sub t ~pos:l ~len:(String.length t - l)
+  with Not_found ->
+    ""
```

Comment:
I think you can use `Filename.extension` from `Import` here instead of writing your own function. Or just not have a wrapper at all. Your call.

Reply:
The hackathon wine had clearly got to me by that point, and I forgot that `Filename` is extended in `import.ml`. I've renamed it to `extension` as an alias (I prefer to have the extra function, because then you don't have to cast the path itself to a string in order to use `Filename.extension`)